docs/src/modules/components/overview/charts/essentialCharts/PieChartDemo.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { pieArcLabelClasses, PieChart } from '@mui/x-charts/PieChart';
+import ChartDemoWrapper from '../ChartDemoWrapper';
+
+// Data derived from https://gs.statcounter.com/os-market-share/desktop/worldwide/2023
+// And https://gs.statcounter.com/os-market-share/mobile/worldwide/2023
+// And https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/worldwide/2023
+// For the month of December 2023
+
+const desktopOS = [
+  {
+    label: 'Windows',
+    value: 72.72,
+  },
+  {
+    label: 'OS X',
+    value: 16.38,
+  },
+  {
+    label: 'Linux',
+    value: 3.83,
+  },
+  {
+    label: 'Chrome OS',
+    value: 2.42,
+  },
+  {
+    label: 'Other',
+    value: 4.65,
+  },
+];
+
+const valueFormatter = (item: { value: number }) => `${item.value}%`;
+
+function Pie() {
+  return (
+    <PieChart
```

Comment:
![image](https://github.com/user-attachments/assets/589bc6bf-0622-4c73-87cb-e63d4a979e57)

Should we add a label for Windows (and maybe OS X)? The chart looks empty to me